### DISCUSSION TITLE
Does the extension support "open this include file"?

### DISCUSSION THREAD

**Discussion by irmackenzie** [microsoft/vscode-cpptools#7004] [Q&A] [answered]
I often use the feature in Visual Studio to right-click and open a #included file in the IDE, but I couldn't find an equivalent in vscode with the c++ extension installed. Is it not supported or did i just fail to Google how to do it? Thanks!

**Comment by sean-mcmanus** [comment, marked as answer]
Go to Definition or Ctrl+Click.

**Reply by irmackenzie** [reply]
Thanks! I had tried this with a header and it gave a "No definition found" error so I assumed go to definition didn't work for headers. However I see it works for other headers.
